src/mem3/src/mem3_reshard.erl

```diff
@@ -265,8 +265,9 @@ handle_call({resume_job, _}, _From, #state{state = stopped} = State) ->
     end;
 handle_call({resume_job, Id}, _From, State) ->
     couch_log:notice("~p resume_job call ~p", [?MODULE, Id]),
-    case job_by_id(Id) of
-        #job{job_state = stopped} = Job ->
+    #job{job_state = JobState} = Job = job_by_id(Id),
+    case JobState of
+        JobState when JobState == stopped; JobState == failed ->
```

Comment:
We should match a #job{} object not just a job state. This accounts for the 
```
**error:{badmatch,{500,
           #{<<"error">> => <<"case_clause">>,
             <<"reason">> =>
                 <<"[{badrpc,\n     {'EXIT',\n         {{{case_clause,complete"...>>,
             <<"ref">> => 3529533974}}}
  output:<<"">>
```

Will put a fixup to the PR